recipes/configs/eleuther_eval.yaml

```diff
@@ -5,19 +5,24 @@
 
 # Model Arguments
 model:
-  _component_: torchtune.models.llama2.llama2_7b
+  _component_: torchtune.models.llama2.llama2_13b
```

Comment:
Out of curiosity, why do we default to 13B now? Without quantization I take it this won't run in <24GB, right?

Reply:
Oh I can revert back